home tasks(1-8)

Olgamaple opened the pull request:
home work 2 - 8

Olgamaple committed:
home tasks(1-8)

mezzija commented on main.js:
```diff
@@ -0,0 +1,300 @@
+
+console.log(sum(5)(2));
+
+const arr5 = [12, 53, 20, 18, 22, 10, 43, 57, 50, 1];
+const func = (array) => {
+  for (let i = 0; i < arr.length; i++) {
+    if (array[i] % 10 === 0) {
+      array[i] = array[i].toString().replaceAll('0', 'zero');
+    }
+  }
+  return array;
```
третий вариант с заменой ?

mezzija commented on main.js:
```diff
@@ -0,0 +1,300 @@
+const colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
+const pGroup = document.querySelectorAll('p');
+const f1 = () => {
+  let k = 0;
```
давай переменные называть именами, которые показывают что они делают 
вот тут например счетчик 
можно назвать переменную `counter`,а если оставить просто k, то кто его знает что она будет делать )))

mezzija commented on main.js:
```diff
@@ -0,0 +1,300 @@
+  let k = 0;
+  return event => {
+    event.target.style.color = colors[k];
+    k++;
+    if (k === colors.length) {
+      k = 0;
+    }
+  };
```
попробуй может тут заменить данные строчки под тернарное выражение )))

mezzija commented on main.js:
```diff
@@ -0,0 +1,300 @@
+console.log(func(arr5));
+
+const colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
+const pGroup = document.querySelectorAll('p');
```
и тут 
селектор назвала просто `p`. А что за pю Что в этом селекторе лежит я не знаю )))

Olgamaple committed:
home tasks(1-8)
rewrote the condition to the ternary expression

rename variables to correct names

mezzija commented on main.js:
```diff
@@ -0,0 +1,286 @@
+
+const colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
+const groupEl = document.querySelectorAll('p');
+const f1 = () => {
```
что означает название функции f1?))))

mezzija commented on main.js:
```diff
@@ -0,0 +1,286 @@
+console.log(sum(5)(2));
+
+const colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
+const groupEl = document.querySelectorAll('p');
```
измени название класса с `p` на какой нибуль другой. Надо чтобы название класса говорило о том что тут происходит

mezzija commented on main.js:
```diff
@@ -0,0 +1,286 @@
+const groupEl = document.querySelectorAll('p');
+const f1 = () => {
+  let counter = 0;
+  return event => {
+    event.target.style.color = colors[counter];
+    counter = counter === colors.length ? 0 : counter;
+  };
```
работает у тебя такое выражение ?

Olgamaple committed:
home tasks(1-8)
rewrote the condition to the ternary expression

rename variables to correct names
changed name function
add plus one in ternary expression
